### DISCUSSION TITLE
Loud Static

### DISCUSSION THREAD

**Discussion by LoafOfQtips** [lightspark/lightspark#932] [General]
No matter if there is sound present in the standalone player, loud static maxes out my headphones. If there is any audio present within the player, it makes a very bassy thud before returning to loud static.

**Comment by dbluelle** [comment]
Hmm, we don't do any fancy stuff with audio, especially if the swf file doesn't have any sound embedded...
So we need more information:
- which OS are you using
- which version of lightspark are you using (output of lightspark -v on the command line)
- please provide an example of an swf file that produces the issue
- please provide the log output of running that swf file (lightsprk -l 4 file.swf)

**Comment by dbluelle** [comment]
I've just fixed some sound issues (see https://github.com/lightspark/lightspark/issues/931) maybe this also fixes your problems.

**Comment by TheGentleChainsaw** [comment]
I'm also encountering loud static whenever a SWF file tries to play audio. For a while, I thought it might be trying to play the audio clips back at the wrong sample rate (this was on the 0.8.5 builds), but I've tried files with 11025Hz, 22050Hz, and 44100Hz audio and they all give me the exact same problem, whether it's through my monitor's headphone jack or the audio output on my motherboard or even through one of my USB soundcards.

Currently running version 0.8.6 on Windows 10.

**Reply by dbluelle** [reply]
As stated above we need  
- an example of an swf file that produces the issue
- the log output of running that swf file (lightspark -l 4 file.swf)

Without that we will be unable to reproduce the issue and can not fix it.

**Reply by TheGentleChainsaw** [reply]
I _was_ going to reiterate that the static problem occurs no matter what file I'm playing, but you're right. I should still include an example with log data that's specific to my system.
[ukigumo.zip](https://github.com/lightspark/lightspark/files/9391303/ukigumo.zip)

**Comment by n0samu** [comment]
This problem happens with every SWF that plays sound on Windows for me. On Linux (Ubuntu) I have no such issue. I've attached an SWF and log below but again, it happens with every SWF.
[bejeweledtwist.zip](https://github.com/lightspark/lightspark/files/9389755/bejeweledtwist.zip)

**Reply by n0samu** [reply]
Just did a bit more testing, and the issue is also present in the 0.8.5 release, but the sound is normal in the 0.8.4.1 release. (I'm on Windows 10)

**Reply by TheGentleChainsaw** [reply]
+1 for this. I just tried the 0.8.4.1 build with some of my locally-saved files (including ukigumo.swf from the zip file that I provided above) and I can confirm they sound exactly like they're supposed to.

**Comment by 6ab18b** [comment]
Same issue as above. The version of Windows I'm using is 8.1 build 9600. The following archive contains a few swfs where the audio occasionally fades in at specific points or seems to be partially present in the background (or at least I think that's the case for Just_Breathe and 7377). cavestory (as an aside, in 0.8.5 release, the preloader on this had no issues but now it does, if you wait for it to pass then you can still hear it) and Ambience in particular are the only ones that momentarily sound as they should.
[static7.zip](https://github.com/lightspark/lightspark/files/9391231/static7.zip)

**Reply by 6ab18b** [reply]
[logs.zip](https://github.com/lightspark/lightspark/files/9398812/logs.zip)

**Comment by dbluelle** [comment]
Fortunately I could reproduce the issue with wine...
I fixed it ( at least for wine ;) ) and made a hotfix release, so it should now work on Windows, too.

**Reply by TheGentleChainsaw** [reply]
Looks like the hotfix worked, at least for me. Thanks for sorting it out!

**Reply by huwdp** [reply]
Thank you and everyone that reported this issue! :)

**Reply by eiglow** [reply]
Ah, good to know that wine can work for testing this stuff.
